.github/workflows/preview.yaml

```diff
@@ -0,0 +1,77 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
```

Comment:
Wrong license header

Reply:
I've picked it from here - https://github.com/apache/camel-website/blob/main/.github/workflows/main-push-ui-bundle.yml

Reply:
Yeah, but you're not licensing this to Apache Software Foundation :) Try something like this: https://github.com/enterprise-contract/ec-cli/blob/main/.github/workflows/checks.yaml